release notes/v0.48.0.md

```diff
@@ -33,13 +33,11 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Bug fixes
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#111` fixes race condition in runtime_
+- [browser#1077](https://github.com/grafana/xk6-browser/pull/1077) Fixes `browserContext.clearPermissions` so it no longer panics and actually performs the clear permissions action in the chrome browser.
```

Comment:
This is OK, but what about the following? Should we also link to the documentation like we did in previous release notes?

```suggestion
- [browser#1077](https://github.com/grafana/xk6-browser/pull/1077) Fixes `browserContext.clearPermissions` to clear permissions without panic.
```

Reply:
I'm not sure we need to link to documentation for fixes like this, I can see value in it if there's a new feature though or a breaking change but i guess it depends on a case-by-case basis. I like the suggestion you've made though 👍

Reply:
Sure, no worries.